redpanda/cloud/ratelimiter.go

```diff
@@ -0,0 +1,94 @@
+package cloud
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"golang.org/x/time/rate"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	limitPeriod = 60.0
+	burstPeriod = 10.0
```

Comment:
please document these variables and why they are 60.0 and 10.0

Reply:
Derived from the rate limit